src/Offer/ReadModel/JSONLD/OfferLDProjector.php

```diff
@@ -100,6 +100,8 @@ abstract class OfferLDProjector implements OrganizerServiceInterface
 
     protected VideoNormalizer $videoNormalizer;
 
+    private array $weights;
```

Comment:
Can we add a docblock here what type of array this is? I assume an array of int?